Clicker/ViewControllers/GroupControlsViewController.swift

```diff
@@ -39,18 +42,23 @@ class GroupControlsViewController: UIViewController {
     let infoViewHeight: CGFloat = 20
     let infoViewTopPadding: CGFloat = 28
     let infoViewHorizontalPadding: CGFloat = 30
-    let collectionViewTopPadding: CGFloat = 16
+    let collectionViewTopPadding: CGFloat = 10
     let collectionViewBottomPadding: CGFloat = 24
     let collectionViewHeight: CGFloat = 218
-    let exportButtonWidthScaleFactor: CGFloat = 0.43
-    let exportButtonHeight: CGFloat = 47
-    let exportButtonBorderWidth: CGFloat = 1
-    let exportButtonTopPadding: CGFloat = 12
-    let navigtionTitle = "Group Controls"
+    let spaceOneHeight: CGFloat = 38
+    let spaceTwoHeight: CGFloat = 16
+    let spaceThreeHeight: CGFloat = 57
+    let navigationTitle = "Group Controls"
     let backImageName = "back"
-    let attendanceLabelText = "Attendance"
-    let exportButtonTitle = "Export"
-
+    let attendanceHeaderLabel = "Attendance"
+    let pollSettingsHeaderLabel = "Poll Settings"
+    let liveQuestionsTitle = "Live Questions"
+    let filterTitle = "Filter Responses"
+    let locationTitle = "Location Restriction"
+    let liveQuestionsDescription = "Allow audience to ask questions to \nhost during session"
+    let filterDescription = "Filter responses to prohibit \ninappropriate language"
+    let locationDescription = "Only allow poll members within 300 \nfeet to participate"
```

Comment:
shouldn't these constants be alphabetized to make it easier to add new ones in the future?